NDB.Covid19/NDB.Covid19/ViewModels/ENDeveloperToolsViewModel.cs

```diff
@@ -245,32 +246,52 @@ public string GetExposureInfosFromLastPull()
+                string result = $"AttenuationThresholds: {EnConfArrayString(dsc.AttenuationThresholds)}\n " +
+                    $"AttenuationWeights: {EnConfDictionaryString(dsc.AttenuationWeights)} \n" +
+                    $"DaysSinceLastExposureThreshold: {dsc.DaysSinceLastExposureThreshold} \n" +
+                    $"DaysSinceOnsetInfectiousness: {EnConfDictionaryString(dsc.DaysSinceOnsetInfectiousness)} \n" +
+                    $"DefaultInfectiousness: {dsc.DefaultInfectiousness}\n" +
+                    $"DefaultReportType: {dsc.DefaultReportType}\n" +
+                    $"InfectiousnessWeights: {EnConfDictionaryString(dsc.InfectiousnessWeights)}\n" +
+                    $"ReportTypeWeights: {EnConfDictionaryString(dsc.ReportTypeWeights)}\n";
+                DevToolsOutput = result;
+            }
+            catch (Exception)
+            {
+                EN.Configuration c = await new ExposureNotificationHandler().GetConfigurationAsync();
+                string res = $" AttenuationWeight: {c.AttenuationWeight}, Values: {EnConfArrayString(c.AttenuationScores)} \n" +
+                    $" DaysSinceLastExposureWeight: {c.DaysSinceLastExposureWeight}, Values: {EnConfArrayString(c.DaysSinceLastExposureScores)} \n" +
+                    $" DurationWeight: {c.DurationWeight}, Values: {EnConfArrayString(c.DurationScores)} \n" +
+                    $" TransmissionWeight: {c.TransmissionWeight}, Values: {EnConfArrayString(c.TransmissionRiskScores)} \n" +
+                    $" MinimumRiskScore: {c.MinimumRiskScore}" +
+                    $" DurationAtAttenuationThresholds: [{c.DurationAtAttenuationThresholds[0]},{c.DurationAtAttenuationThresholds[1]}]";
+                Debug.WriteLine("Exposure Configuration:");
+                Debug.WriteLine($" AttenuationWeight: {c.AttenuationWeight}, Values: {EnConfArrayString(c.AttenuationScores)}");
+                Debug.WriteLine($" DaysSinceLastExposureWeight: {c.DaysSinceLastExposureWeight}, Values: {EnConfArrayString(c.DaysSinceLastExposureScores)}");
+                Debug.WriteLine($" DurationWeight: {c.DurationWeight}, Values: {EnConfArrayString(c.DurationScores)}");
+                Debug.WriteLine($" TransmissionWeight: {c.TransmissionWeight}, Values: {EnConfArrayString(c.TransmissionRiskScores)}");
+                Debug.WriteLine($" MinimumRiskScore: {c.MinimumRiskScore}");
+                DevToolsOutput = res;
+            }
+
             DevToolUpdateOutput?.Invoke();
-            await _clipboard.SetTextAsync(res);
-            return res;
+            await _clipboard.SetTextAsync(DevToolsOutput);
+            return DevToolsOutput;
+        }
+
+        private string EnConfDictionaryString<TKey, TValue>(IDictionary<TKey, TValue> dictionary)
+        {
+            return "{" + string.Join(",", dictionary.Select(kv => kv.Key + "=" + kv.Value).ToArray()) + "}";
         }
 
         private string EnConfArrayString(int[] values)
```

Comment:
Rename to ENConfArrayString, otherwise, looks like English :)

Reply:
done